src/seedu/addressbook/data/person/Person.java

```diff
@@ -10,10 +10,16 @@
  */
 public class Person implements ReadOnlyPerson {
 
+    /**
+     * A sequence number to keep track of sequence in which different persons are added
+     */
+    public static int nextSequenceNumber = 0;
```

Comment:
Should this be 0 or 1?

Reply:
Oh it's 1. As it is a 1-based index.
Thank you!